osu.Game/Configuration/ToolbarClockDisplayMode.cs

```diff
@@ -7,7 +7,10 @@ public enum ToolbarClockDisplayMode
     {
         Analog,
         Digital,
+        Digital12H,
         DigitalWithRuntime,
-        Full
+        DigitalWithRuntime12H,
+        Full,
+        Full12H
```

Comment:
really unsure about this. generally why we ask you consult before opening a PR.

i'd sooner have the 12 hour mode based off system settings than add even more overloads to this enum.